### DISCUSSION TITLE
Devices Within A Physical Chassis

### DISCUSSION THREAD

**Discussion by JosephLive93** [netbox-community/netbox#16682] [Q&A]
Is there a way to pair two devices into a physical chassis? This becomes a little difficult because the physical chassis is not serialized or doesn't have a specific name to it except the model of the chassis. I am trying to build out hundreds of these within Netbox and I'm not sure the best way to do this.

Would this be used with the 'Device Bays' portion? Because this requires a parent device and child device, however the chassis itself isn't necessarily a 'device' but rather a tray that holds two devices. This becomes an issue when I'm trying to put two devices on this tray in the same position of a rack.

Thanks for any help!

**Comment by jsenecal** [comment]
You do need a "parent" device, a shelf, in your example with multiple device-bays.

You then add child devices with their own components to it.

**Reply by JosephLive93** [reply]
From the perspective of the rack view, can you view at the level of of the device bay and not the device itself? For example, viewing the front of a rack and putting one device in a device bay on the left and one device in the device bay on the right

**Reply by candlerb** [reply]
You can give the parent device an image, but it will be the same regardless of whether the device bays are populated or not. The child device image is not used AFAIK.
